test/functional/wallet_spend_unconfirmed.py

```diff
@@ -0,0 +1,436 @@
+        assert_greater_than_or_equal(resulting_fee_rate, self.target_fee_rate)
+
+    def test_target_feerate_confirmed(self):
+        self.log.info("Start test feerate with confirmed input")
+        wallet = self.setup_and_fund_wallet("confirmed_wallet")
+
+        ancestor_aware_txid = wallet.sendtoaddress(address=self.def_wallet.getnewaddress(), amount=0.5, fee_rate=self.target_fee_rate)
+        ancestor_aware_tx = wallet.gettransaction(txid=ancestor_aware_txid, verbose=True)
+        self.assert_beats_target(ancestor_aware_tx)
+
+        self.generate(self.nodes[0], 1)
+        wallet.unloadwallet()
+
+    def test_target_feerate_unconfirmed_high(self):
+        self.log.info("Start test feerate with high feerate unconfirmed input")
+        wallet = self.setup_and_fund_wallet("unconfirmed_high_wallet")
+
+        # Send unconfirmed transaction with high feerate to testing wallet
+        parent_txid = wallet.sendtoaddress(address=wallet.getnewaddress(), amount=1, fee_rate=100)
+        parent_tx = wallet.gettransaction(txid=parent_txid, verbose=True)
+        self.assert_beats_target(parent_tx)
+
+        ancestor_aware_txid = wallet.sendtoaddress(address=self.def_wallet.getnewaddress(), amount=0.5, fee_rate=self.target_fee_rate)
+        ancestor_aware_tx = wallet.gettransaction(txid=ancestor_aware_txid, verbose=True)
+
+        self.assert_spends_only_parent(ancestor_aware_tx, parent_txid)
+
+        self.assert_beats_target(ancestor_aware_tx)
+        resulting_ancestry_fee_rate = self.calc_set_fee_rate([parent_tx, ancestor_aware_tx])
+        assert_greater_than_or_equal(resulting_ancestry_fee_rate, self.target_fee_rate)
+
+        self.generate(self.nodes[0], 1)
+        wallet.unloadwallet()
+
+    def test_target_feerate_unconfirmed_low(self):
+        self.log.info("Start test feerate with low feerate unconfirmed input")
+        wallet = self.setup_and_fund_wallet("unconfirmed_low_wallet")
+
+        parent_txid = wallet.sendtoaddress(address=wallet.getnewaddress(), amount=1, fee_rate=1)
+        parent_tx = wallet.gettransaction(txid=parent_txid, verbose=True)
+
+        self.assert_undershoots_target(parent_tx)
+        resulting_fee_rate_funding = self.calc_fee_rate(parent_tx)
+        assert_greater_than(self.target_fee_rate, resulting_fee_rate_funding)
+
+        ancestor_aware_txid = wallet.sendtoaddress(address=self.def_wallet.getnewaddress(), amount=0.5, fee_rate=self.target_fee_rate)
+        ancestor_aware_tx = wallet.gettransaction(txid=ancestor_aware_txid, verbose=True)
+
+        self.assert_spends_only_parent(ancestor_aware_tx, parent_txid)
+
+        self.assert_beats_target(ancestor_aware_tx)
+        resulting_ancestry_fee_rate = self.calc_set_fee_rate([parent_tx, ancestor_aware_tx])
+        assert_greater_than_or_equal(resulting_ancestry_fee_rate, self.target_fee_rate)
+        assert_greater_than_or_equal(self.target_fee_rate*1.5, resulting_ancestry_fee_rate)
```

Comment:
Where does the 1.5 come from here (and in later tests e.g. `test_two_low_feerate_unconfirmed_parents()` 1.1)?